.gitignore

```diff
@@ -0,0 +1,6 @@
+# NOTE: we don't include OS- or IDE/editor-specific files (e.g. Windows' Thumbs.db,
+# OSX's .DS_Store, Vim's *.swp, Emacs' *~, Visual Studio Code's .vscode, etc.) on-purpose.
+# Those patterns should go in a global gitignore or repository-specific excludes.
+# See https://help.github.com/articles/ignoring-files/ for how to configure your environment.
+# See https://github.com/github/gitignore/tree/master/Global for a list of global ignore rules.
+/bazel-*
```

Comment:
This should be part of another PR as it's not related to TravisCI

Reply:
Fixed - moved to https://github.com/google/jsinterop-generator/pull/14